packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
+                description: Contract of the NFT
+                example: 0x57f1887a8bf19b14fc0d912b9b2acc9af147ea85
+              token:
+                type: number
+                description: Token id of the NFT
+                example: 123
+
+    task:
+      type: object
+      table: task
+      properties:
+        id:
+          type: string
+          description: Task ID
+          example: 1-23-123-123-1123-123
+        kind:
+          type: string
+          description: Type of the task
+          example: import
+        status:
+          type: string
+          description: Status of the task
+          example: ready
+        createdAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was created
+          example: 2022-01-01T02:20:00Z
+        updatedAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was updated
+          example: 2022-01-01T02:20:00Z
+        params:
+          type: object
+          description: Parameters of the task
+          properties:
+            import:
+              type: object
+              description: Parameters for the import task
+              properties:
+                url:
+                  type: string
+                  description: URL of the asset to import
+                  example: https://cdn.livepeer.com/ABC123/filename.mp4
+            export:
+              type: object
+              description: Parameters for the export task
+              properties:
+                kind:
+                  type: string
+                  description: Type of the export
+                  example: arweave|ipfs|s3
+                s3:
+                  type: object
+                  description: Parameters for the S3 export
+                  properties:
+                    destination:
+                      type: string
+                      description: Destination of the export
+                      example: s3://bucket/path/to/file.mp4
+                    key:
+                      type: string
+                      description: Key of the export
+                      example: ABC123
+                    secret:
+                      type: string
+                      description: Secret of the export
+                      example: ABC123
+            transcode:
+              type: object
+              description: Parameters for the transcode task
+              properties:
+                assetId:
+                  type: string
+                  description: ID of the asset to transcode
+                  example: 123123123123
+                preset:
+                  type: string
+                  description: Preset to use for the transcode
+                  example: 1080p
+                container:
+                  type: string
+                  description: Container to use for the transcode
+                  example: mp4
+                audio:
```

Comment:
Is a transcoding task with audio specification a muted output or a `acodec copy`? Should we specifiy a `muted` param in the audio parameters?

Reply:
Let's mimic the `cli-transcoder` for the MVP! If it doesn't support muting audio we might want to add the feature later. But I think it (and Livepeer) does not support audio-transcoding right now, so it will be unlikely that we support it **soon** (might support it sometime tho)